lib/winston-logstash-udp.js

```diff
@@ -79,6 +80,11 @@ LogstashUDP.prototype.log = function(level, msg, meta, callback) {
 
 LogstashUDP.prototype.sendLog = function(message, callback) {
     var self = this;
+
+    if (this.trailingLineFeed === true) {
+        message = message.replace(/\n$/, '') + "\n";
+    }
+
```

Comment:
In this case, I think there should also be an option for `trailingLineFeedChar` (or something like that) that defaults to `os.EOL` and use that option instead of hard coding `\n`